mobify/hijax
# Release v0.2.1

Release v0.2.1

## Changes
- Fix bug with response headers not being cleared and reset for every request
- Some better build tasks + documentation in README
- Tidying up of dependencies (bower and npm)

## How To Test
- Checkout code
- `npm i`
- `./node_modules/.bin/bower install`
- Run Hijax-specific tests with `npm t`
- Run `npm link`
- In a project that uses Hijax, run `npm link hijax`
- Make sure project code uses Hijax through NPM
- Test Hijax in the project

## Related
- https://github.com/mobify/extra-mobile/pull/769

## TODOs:
- [x] Eng +1
